lstchain/mc/sensitivity.py

```diff
@@ -62,7 +62,7 @@ def process_mc(simtelfile, dl2_file):
     events = pd.read_hdf(dl2_file)
 
     e_reco = 10**events.mc_energy * u.GeV
-    n_reco = e_reco.shape[0]
+    # n_reco = e_reco.shape[0]
     gammaness = events.gammaness
     theta2 = (events.src_x - events.src_x_rec)**2 + \
         (events.src_y - events.src_y_rec)**2 * u.deg**2
```

Comment:
If you prefer to change the theta2 definition, please do so

Reply:
Will do in a future PR.
I'd like to introduce the use of other libraries for common stuff like this.

Reply:
I just wanted to make a quick fix here.

Reply:
which other libraries? 
well we can discuss about this later, but I would try not to make it too complex if the computing time overhead is too large.

Reply:
We are here trying to re-implement many things that are already implemented in `astropy`, `gammapy` or `ctaplot` which means maintenance that we should avoid (and potentially also means wrong implementations).
But we can discuss that later.